### PR TITLE
Export document bugfixes

### PR DESCRIPTION
**Pull request by AbdusShaikh**
This PR addresses requested changes to [LF-2322](https://lite-farm.atlassian.net/browse/LF-2322).

See [PR #1929 ](https://github.com/LiteFarmOrg/LiteFarm/pull/1929) for instructions on how to test this. 

With these changes, a PDF of the survey questions should not be included in the final export, and the xlsx document should include the relevant farm id and survey id in the header. Additionally, questions nested inside multiple groups/pages should export correctly now.

### CODE REVIEW

File: packages/api/src/jobs/certification/survey_record.js [hunk 1]
```diff
@@ -65,15 +65,28 @@ module.exports = async (submission, exportId) => {
...
+    for (const group of parentGroups) {
+      if (group != null) {
+        answer = answer[group];
+      }
+    }
+    return answer[name]?.value;
+  };
+
+  const getQuestionInfo = (questionAnswerList, groupName = null, prevParentGroups = []) => {
+    console.log('PARENT GROUPS: ', prevParentGroups);
```
Comment (Cali0707) [inline packages/api/src/jobs/certification/survey_record.js:83]: We should remove this console log

File: packages/api/src/jobs/certification/survey_record.js [hunk 2]
```diff
@@ -262,7 +275,8 @@ module.exports = async (submission, exportId) => {
       .cell(`${col}${row}`)
       .value(data['Question'])
       .style({ ...groupHeaderStyle, ...getGroupBorder('top') });
-    const childInfo = getQuestionInfo(data['Children'], data['Name']);
+    const childInfo = getQuestionInfo(data['Children'], data['Name'], data['ParentGroups']);
+    console.log('CHILDREN: ', childInfo);
```
Comment (Cali0707) [inline packages/api/src/jobs/certification/survey_record.js:279]: We should remove this console log

**Review by Cali0707 — CHANGES_REQUESTED** [review]
@AbdusShaikh this PR seems to be having the same failure we were experiencing yesterday regarding exports. To recreate: set the `survey_id` of the certifier you are using on your farm to `null`. Then, generate a certification export. The job will fail.

Besides this, the other changes seem to be working!

**Review by Cali0707 — APPROVED** [review]
This works now 👍

**Review by smattingly — APPROVED** [review]

**Pull request merged**